src/SprykerEco/Zed/AfterPay/Business/AfterPayBusinessFactory.php

```diff
@@ -318,7 +318,7 @@ public function createAuthorizeRequestBuilder(): AuthorizeRequestBuilderInterfac
         $authorizeRequestBuilderStack = $this->getAuthorizeRequestBuilderStack();
 
         if (!isset($authorizeRequestBuilderStack[$authorizeWorkflow])) {
-            throw new InvalidAfterPayAuthorizeRequestBuilderException($authorizeWorkflow . " is not a valid AfterPay authorize request builder.");
+            throw new InvalidAfterPayAuthorizeRequestBuilderException($authorizeWorkflow . ' is not a valid AfterPay authorize request builder.');
```

Comment:
I would suggest to use `sprintf` here and below

Reply:
Done